standards/open-to-contributions.md

```diff
@@ -24,16 +25,31 @@ order: 4
 
 ## What this doesn’t do
 
-* Get others to reuse the codebase
+* Guarantee others will reuse the codebase
 
 ## How to test
+* There's a public issue tracker.
+* It's possible to submit suggestions for changes to the codebase.
+* There are contribution guidelines.
 
 ## Policy makers: what you need to do
+* Add a list to the codebase of any other resources policy experts, non-governmental organisations and academics would find useful for understanding and reusing your policy.
+* Track policy issues in the codebase, so that a relevant external policy expert can volunteer help.
+* Consider adding contact details so that other policy makers considering reuse can ask you for advice.
 
 ## Management: what you need to do
 
 * Add to the documentation: how your organisation is involved in the project, what resources it has available for it and for how long
+* Track management issues in the codebase, so that external managers with relevant experience can volunteer help.
+* Support your experienced policy makers, developers and designers to keep contributing to the codebase for as long as possible.
 
 ## Developers and designers: what you need to do
+* Respond promptly to requests
+* Be welcoming and teach your colleagues about open source culture
 
 ## Further reading
+* [Building welcoming communities](https://opensource.guide/building-community/)
+* an example [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct) you could include in your codebase
```

Comment:
The Contributor Convenant is a Code of conduct for those who might wonder. Some context might be useful. 

Perhaps this is a nice `MAY` to add (or SHOULD even).

Reply:
I've made a it a MAY for now.